.github/workflows/release-nightly.yml

```diff
@@ -41,10 +41,20 @@ jobs:
         if: steps.cache-deps.outputs.cache-hit == 'true'
       # </common-build>
 
+      - name: Get version
+        id: version
+        run: |
+          PACKAGE_VERSION=$(node -p "require('./packages/lodestar/package.json').version")
```

Comment:
looks like this command returns incorrect value as in https://github.com/ChainSafe/lodestar/runs/6338092886?check_suite_focus=true

```
Run VERSION=$(node -p "require('./packages/lodestar/package.json').version")
VERSION 0.34.0-dev.173+28e2c74cf0
```

so I'm afraid this time it'd get the wrong value too, can we test the output of this step in this branch?

Reply:
Should be fine because we run this now before lerna canary

Reply:
Why not take version from https://github.com/ChainSafe/lodestar/blob/master/lerna.json#L7 ?